grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ArmeriaClientCall.java

```diff
@@ -490,7 +490,7 @@ private void close(Status status, Metadata metadata) {
         }
 
         final RequestLogBuilder logBuilder = ctx.logBuilder();
-        logBuilder.responseContent(GrpcLogUtil.rpcResponse(status, firstResponse), null);
+        logBuilder.responseContent(GrpcLogUtil.rpcResponse(status, firstResponse, metadata), null);
         if (status.isOk()) {
             req.abort();
         } else {
```

Comment:
We also need to use `StatusException` here instead of `StatusRuntimeException`.
https://github.com/grpc/grpc-java/issues/4683

Reply:
It's another issue but fixed in https://github.com/line/armeria/pull/4636/commits/71334a6de5bd678c13d178b882dfeb45fe6db779.
I didn't find the code which rely on StatusRuntimeException type, but since users can subscribe `ctx.request()`, it can be considered as minor breaking change?

Reply:
> It's another issue but fixed in https://github.com/line/armeria/commit/71334a6de5bd678c13d178b882dfeb45fe6db779.

Yes, my comment was related to this: https://github.com/line/armeria/pull/4636#discussion_r1092798637
😄 

Sorry for making you get work twice but let's roll it back and apply it in the follow-up PR.

Reply:
Because this PR is merged, I think we need to handle https://github.com/line/armeria/pull/4636#discussion_r1092798637 before the next release. Are you going to send a PR for that? If not, let us take it. 😄 Just let me know which way you want.

Reply:
> Are you going to send a PR for that? If not, let us take it. 😄 Just let me know which way you want.

Oops,was missing it..
Can you take it? 🙏